### DISCUSSION TITLE
Adding eslint config

### DISCUSSION THREAD

**Discussion by beonma** [usebruno/bruno#726] [Ideas]
I've been actively working on the project and relied on numerous console logs for debugging. When attempting to push my code, I realized the painstaking process of manually removing each console log and ensuring none were left behind. I believe it's time to implement ESLint with a minimal configuration that we can expand upon in the future. What are your thoughts on this idea?

**Comment by grubersjoe** [comment]
I think this a good idea and a project of this size should have a linter 👍. Also starting small and iterating on it sounds like a good approach. A somewhat related discussion: https://github.com/usebruno/bruno/discussions/675
